demo/db_namer.py

```diff
@@ -0,0 +1,87 @@
+"""Utility function for generating database names from session ID cookies."""
+
+ADJECTIVES = [
```

Comment:
I think we should choose neutral words here (avoid things like `stupid`, `hideous`, `grisly`) and also avoid anything that could potentially represent an ethnicity (`asian`, `black`, `white`) because we could end up randomly generating unpleasant combinations. Perhaps including or entirely using nouns might make it easier (stuff like `car` or `fox` or `airplane`)?